src/resources/generated/issuing_card.rs

```diff
@@ -737,7 +736,6 @@ pub enum IssuingCardSpendingLimitCategories {
     TruckUtilityTrailerRentals,
     TypesettingPlateMakingAndRelatedServices,
     TypewriterStores,
-    #[serde(rename = "u_s_federal_government_agencies_or_departments")]
     USFederalGovernmentAgenciesOrDepartments,
```

Comment:
Same here. u_s_federal_government_agencies_or_departments.

Reply:
Same as already answered, this rename is just not needed.